Pkgfile

```diff
@@ -5,7 +5,7 @@ format: v1alpha2
 vars:
   LINUX_FIRMWARE_VERSION: "20240410" # update this when updating PKGS_VERSION in Makefile
   DRBD_DRIVER_VERSION: 9.2.8 # update this when updating PKGS_VERSION in Makefile
-  ZFS_DRIVER_VERSION: 2.2.3 # update this when updating PKGS_VERSION in Makefile
+  ZFS_DRIVER_VERSION: 2.2.4 # update this when updating PKGS_VERSION in Makefile
```

Comment:
the checksums also needs to be updated at https://github.com/siderolabs/extensions/blob/main/storage/zfs/zfs-tools/pkg.yaml#L14, also this needs updating first https://github.com/siderolabs/pkgs/blob/main/Pkgfile#L173

Reply:
https://github.com/siderolabs/pkgs/pull/952